src/ed_object_models/conversion_sdf.py

```diff
@@ -389,20 +416,19 @@ def convert_world(yml, model_name, recursive=False):
     world_include = world["include"]
     world_model = world["model"]
     if not isinstance(yml["composition"], list):
-        print(bcolors.FAIL + bcolors.BOLD + "[{}] composition should be a list".format(model_name) + bcolors.ENDC)
-        return 1
+        raise Exception(bcolors.FAIL + bcolors.BOLD + "[{}] composition should be a list".format(model_name)
```

Comment:
Why keep these bcolors thingies if you're raising? (BTW: I like the raising instead of returning an int that no-one will check